v2/README.md

```diff
@@ -50,16 +47,20 @@ import (
 )
 
 func main() {
-	oauthClientID := os.Getenv("TAILSCALE_OAUTH_CLIENT_ID")
-	tailnet := os.Getenv("TAILSCALE_OAUTH_CLIENT_SECRET")
+	oauthClientID := 
+	oauthClientID := 
 	oauthScopes := []string{"all:write"}
+	tailnet := os.Getenv("TAILSCALE_TAILNET")
 
 	&tsclient.Client{
-		APIKey:    apiKey,
-		Tailnet:   tailnet,
+		Tailnet: os.Getenv("TAILSCALE_TAILNET"),
+		HTTP:    OAuthConfig{
```

Comment:
OAuthConfig needs to be tsclient.OAuthConfig{, no?

(aside: forcing users to rename the package on import isn't ideal... maybe we should add a directory component in there that matches the package name to be idiomatic?)